pkg/kstatus/polling/clusterreader/caching_reader.go

```diff
@@ -5,10 +5,11 @@ package clusterreader
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"sync"
 
-	"k8s.io/apimachinery/pkg/api/errors"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
we use `apierrors` as the alias elsewhere

Reply:
Fixed.